admin_requests.go

```diff
@@ -35,7 +35,65 @@ func (c *Client) EditorToken(runnable *Runnable) (*http.Response, error) {
 			"failed to getEditorTokenFor Runnable: [%s]", runnable.Path())
 	}
 
-	runnable.token = token.Token
+	runnable.editorToken = token.Token
 
 	return res, nil
 }
+
+func (c *Client) UserFunctions(customerID string, namespace string) ([]*Runnable, *http.Response, error) {
+	// GET /api/v1/functions/{customerID}/{namespace}
```

Comment:
delete, or did you want the client to know this is the route?

Reply:
Do you mean delete the comment? It's handy just for quick reference as I'm writing it, and it's not a secret. What I'll probably end up doing is make a nice "header" comment for each function. `go doc` reads those and generates docs from them, so it'll probably include that URL.